haproxy/haproxy.pre-install

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
If an `haproxy` user must be created, this pre-install script should be specified as a scriptlet in the melange yaml file.

Reply:
can you take another look. i'm not sure how melange scriptlets work since there're many examples.